pocket/chain.json

```
@@ -0,0 +1,108 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "pocket",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://www.pokt.network/",
+  "pretty_name": "Pocket Network MainNet",
+  "chain_type": "cosmos",
+  "chain_id": "pocket",
+  "bech32_prefix": "pokt@",
+  "daemon_name": "pocketd",
+  "node_home": "$HOME/.pocket",
+  "key_algos": ["secp256k1"],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "upokt",
+        "fixed_min_gas_price": 0.001,
+        "low_gas_price": 0.001,
+        "average_gas_price": 0.002,
+        "high_gas_price": 0.0025
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "upokt"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/pokt-network/poktroll",
+    "recommended_version": "v0.1.1",
+    "compatible_versions": ["v0.1.1"],
+    "binaries": {
+      "linux/amd64": "https://github.com/pokt-network/poktroll/releases/download/v0.1.1/pocket_darwin_amd64.tar.gz",
+      "linux/arm64": "https://github.com/pokt-network/poktroll/releases/download/v0.1.1/pocket_linux_arm64.tar.gz",
+      "darwin/amd64": "https://github.com/pokt-network/poktroll/releases/download/v0.1.1/pocket_darwin_amd64.tar.gz",
+      "darwin/arm64": "https://github.com/pokt-network/poktroll/releases/download/v0.1.1/pocket_darwin_arm64.tar.gz"
+    },
+    "consensus": {
+      "type": "cometbft",
+      "version": "v0.38.12"
+    },
+    "genesis": {
+      "name": "v0.1.1",
+      "genesis_url": "https://github.com/pokt-network/pocket-network-genesis/blob/master/shannon/testnet/genesis.json"
```

```suggestion
      "genesis_url": "https://raw.githubusercontent.com/pokt-network/pocket-network-genesis/refs/heads/master/shannon/mainnet/genesis.json"
```

This is a mainnet link - assuming this is what we want - a little confused about this as the `network_type` shows `testnet` currently.